version.txt

```diff
@@ -1 +1 @@
-0.1.5
```

Comment:
Thanks for remembering to do this. :yum:

Reply:
We need to automate this part... or at least have a CI check to assert when we do break API... something maybe @xwang233 would be interested?!

Reply:
Oh that's a good idea.

Reply:
I'll check pytorch's CI implementation on detecting BC breaking changes. 😉